lib/rubocop/cop/style/copyright.rb

```diff
@@ -0,0 +1,90 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module Style
+      # Check that a copyright notice was given in each source file.
+      #
+      # The default regexp for an acceptable copyright notice can be found in
+      # config/default.yml.  The default can be changed as follows:
+      #
+      # Style/Copyright:
+      #   Notice: '^Copyright (\(c\) )?2\d{3} Acme Inc'
+      #
+      # This regex string is treated as an unanchored regex.  For each file
+      # that RuboCop scans, a comment that matches this regex must be found or
+      # an offense is reported.
+      #
+      class Copyright < Cop
+        def message
+          "Include a copyright notice matching /#{notice}/" \
+          'before any code.'
+        end
+
+        def notice
+          cop_config['Notice'].to_s
```

Comment:
Agreed, since I am providing defaults for Notice and AutocorrectNotice, I can confidently remove these checks.

My intent was to ensure that I don't have to have logic to check for nil in multiple other places.  To this code, an empty string is eqiuvalent to not specifying something.